lib/snyk.js

```diff
@@ -3,6 +3,7 @@
 const request = require('request-promise-native')
 
 const baseUrl = 'https://snyk.io/api/v1'
+const baseRestUrl = 'https://api.snyk.io'
```

Comment:
As far as I can see, the old `const` isn't used anymore? Why not simply change its value to the new host?

```suggestion
const baseUrl = 'https://api.snyk.io'
```

Reply:
Not all v1 APIs are deprecated. It's still used in the `issues` and `orgs` functions. I will rename the const though, since we have 2 base paths now.